server/utils/match/index.test.ts

```diff
@@ -91,7 +91,6 @@ describe('matchUtils', () => {
         durationInDays: Number(uiParams.durationInDays),
         requirements: {
           apTypes: [uiParams.requirements.apType],
```

Comment:
The `apTypes` property is no longer part of the space search params -- anything to do with those should ideally be cleaned up